tests/unit/src/test/scala/tests/WorkspaceSymbolLspSuite.scala

```diff
@@ -143,9 +143,8 @@ class WorkspaceSymbolLspSuite extends BaseLspSuite("workspace-symbol") {
       )
       _ <- server.didOpen("a/src/main/scala/a/A.scala")
       _ = server.workspaceSymbol("scala.None")
-      option = ".metals/readonly/scala/Option.scala"
-      _ <- server.didOpen(option)
-      references <- server.references(option, "object None")
+      _ <- server.didOpen(".metals/readonly/scala/Option.scala")
+      references <- server.references("a/src/main/scala/a/A.scala", " None")
```

Comment:
So actually, I think this was testing the wrong thing. We should not find references in readonly files when looking in files in the workspace, but it should find them in the file itself.